app/jobs/sync/zendesk/ticket_comments_to_notes.rb

```diff
@@ -0,0 +1,55 @@
+class Sync::Zendesk::TicketCommentsToNotes
+  include Sidekiq::Worker
+
+  # https://developer.zendesk.com/rest_api/docs/support/ticket_comments#list-comments
+  def perform(zendesk_ticket_id, page_number = 0)
+    Rails.logger.info "Starting zendesk ticket #{zendesk_ticket_id}"
+    require 'zendesk_api'
+    client = ZendeskAPI::Client.new do |config|
+      # Mandatory:
+      config.url = "#{Rails.application.secrets[:zendesk_url]}/api/v2" # e.g. https://mydesk.zendesk.com/api/v2
+
+      # Basic / Token Authentication
+      config.username = Rails.application.secrets[:zendesk_username]
+
+      # Choose one of the following depending on your authentication choice
+      # config.token = "your zendesk token"
+      # config.password = "your zendesk password"
+
+      # OAuth Authentication
+      config.access_token = Rails.application.secrets[:zendesk_access_token]
+
+      # Optional:
+
+      # Retry uses middleware to notify the user
+      # when hitting the rate limit, sleep automatically,
+      # then retry the request.
+      config.retry = true
+    end
+
+    publisher_notes = []
+    publisher = nil
+    admin_id = Publisher.find_by(email: Rails.application.secrets[:zendesk_admin_email]).id
+    zendesk_comments = client.ticket.find(id: zendesk_ticket_id).comments
+    for index in (0...zendesk_comments.count)
+      comment = zendesk_comments[index]
+      publisher_email = comment&.via&.source&.from&.address
+
+      if publisher_email.present? && publisher.nil?
+        publisher = Publisher.find_by(email: publisher_email)
+      end
+      publisher_notes << PublisherNote.new(note: comment.plain_body, zendesk_ticket_id: zendesk_ticket_id, zendesk_comment_id: comment.id, created_at: comment.created_at)
```

Comment:
We'll be able to support html in the notes section once https://github.com/brave-intl/publishers/pull/1896 is merged. I don't know if that influences our choice to use the `plain_body` method.